src/backend-test/index.ts

```diff
@@ -0,0 +1,241 @@
+  readonly reportTargetAuthHeaderName?: string
+
+  constructor(options: OttofellerBackendTestProjectOptions) {
+    super({
+      ...options,
+      bundlerOptions: {},
+      jest: false,
+      eslint: false,
+      projenrcTs: true,
+      projenrcJs: false,
+      name: 'backend-test',
+      packageManager: options.packageManager ?? NodePackageManager.NPM,
+      srcdir: options.srcdir ?? '.',
+
+      tsconfig: {
+        compilerOptions: {
+          declaration: undefined,
+          experimentalDecorators: undefined,
+          inlineSourceMap: undefined,
+          inlineSources: undefined,
+          noEmitOnError: undefined,
+          noUnusedLocals: undefined,
+          lib: undefined,
+          outDir: undefined,
+          rootDir: undefined,
+          stripInternal: undefined,
+          forceConsistentCasingInFileNames: true,
+          isolatedModules: false,
+          module: 'esnext',
+          moduleResolution: TypeScriptModuleResolution.NODE,
+          noEmit: true,
+          noUncheckedIndexedAccess: true,
+          baseUrl: './',
+          target: 'es6',
+          skipLibCheck: true,
+          paths: {
+            '*': ['./*'],
+          },
+        },
+      },
+
+      // In case Github is enabled remove all default stuff.
+      githubOptions: {mergify: false, pullRequestLint: false},
+      buildWorkflow: false,
+      release: false,
+      depsUpgrade: false,
+      pullRequestTemplate: false,
+      deps: [
+        'dotenv',
+        'axios',
+        'jest',
+        'ts-jest',
+        'jest-extended',
+        'prettier',
+        'eslint-plugin-prettier',
+        'eslint-config-prettier',
+      ],
+    })
+
+    // ANCHOR Source code
+    const assetsDir = path.join(__dirname, '..', '..', 'src/backend-test/assets')
+    sampleCode(this, options, assetsDir)
+
+    /*
+     * Clean off the projen tasks and if needed replace them with regular npm scripts.
+     * This way we ensure smooth ejection experience with all the commands visible in package.json
+     * and no need to keep the projen task runner within an ejected project.
+     */
+    const tasksToRemove = [
+      'build',
+      'bump',
+      'clobber',
+      'compile',
+      'package',
+      'post-compile',
+      'post-upgrade',
+      'pre-compile',
+      'release',
+      'test',
+      // eslint-disable-next-line @cspell/spellchecker -- the word is used once, so no need to add it to the dictionary
+      'unbump',
+      'upgrade',
+      'watch',
+      'projen',
+    ]
+
+    tasksToRemove.forEach(this.removeTask.bind(this))
+
+    // ANCHOR Jest setup
+    this.addScripts({
+      test: 'jest --detectOpenHandles',
+    })
+
+    new AssetFile(this, 'jest.config.ts', {
+      sourcePath: path.join(assetsDir, 'jest.config.ts'),
+      readonly: false,
+      marker: false,
+    })
+
+    // ANCHOR ESLint and prettier setup
+
+    const lintPaths = options.lintPaths ?? ['.projenrc.ts', '.']
```

Comment:
Why wouldn't you include explicitly the patterns you want to check and instead includes everything? I'm not sure we should use the `format` command on all the projen-generated files. For now I did not find any conflicts when the command is run. Though there is no guarantee for the future, as we can change/add our rules which could contradict what `projen` renders.

Reply:
Sorry, it's not completely clear to me what you mean by including explicitly the patterns. Currently, I am not passing the projenrc file to the addLinters and just using [.] it will be enough?

Reply:
By `include explicitly the patterns you want` I mean getting rid of `.` and manually list the files and folders you want to format/check. Thus you would have the same list defined for both `eslint` and `prettier`. On the other hand with `.` you have all files processed (even the readme) and if anything needs be excluded one must handle separate ignore lists for `prettier` and `eslint`.

However I don't insist. If that's what you expect from the project, then I'm ok with it. Maybe @bdoof has any comment on it.